internal/js_library/v2/package.json

```diff
@@ -3,6 +3,8 @@
     "devDependencies": {
         "@babel/core": "^7.1.0",
         "@babel/preset-env": "^7.1.0",
+        "babel-plugin-transform-es2015-modules-commonjs": "^6.26.2",
```

Comment:
These 6.x plugins work with 7.x? Do they not have 7.x equivalents? Also adding it here means they get added for everyone, I think a solution would be nicer to have that as separate deps just for the example. But I think it is not supported to have deps from separate node_modules, is it?

Reply:
hmm, that's a good question. Honestly I just copied this from my code, but I guess it does work since it's used in an example.

Yeah, the issue is that you can't have deps from separate node_modules. I don't mind removing this though, I can probably find the 7.x equivalent. It's just for the example anyways.

Reply:
I guess that reduces the benefits of `babel_executable` a bit. Maybe it makes sense to have to provide all deps if it is being used? So then at least custom plugins etc can be provided. And yeah I would prefer not having these plugins in the default ones. I might just make the changes to get this merged.

Reply:
Yeah. It reduces the benefits of babel_executable a lot. Let's just remove babel_executable then, it makes it a little unclear what's happening. Users can just make their own babel using the `babel.js` file themselves if they want. I'll do this real quick.